app/src/main/java/io/pslab/activity/LuxMeterActivity.java

```diff
@@ -133,4 +133,4 @@ private int getValueFromText(String strValue, int lowerBound, int upperBound) {
         else if (value < lowerBound) return lowerBound;
         else return value;
     }
-}
+}
```

Comment:
Hi! Please do not make any modifications to files other than the ones addressed in the issue. Please revert this new line deletion entry.

Reply:
@CloudyPadmal I apologize, it might have happened while I was resolving the merge conflicts. Also could you please tell me what to do as the curly brace was on line 136 and is still on line 136 and the code is reformatted, so what should I do ??

Reply:
This change relates to a deletion of a new line.

Reply:
@CloudyPadmal I've reverted the changes, however, I tried but I  was not able to squash the
commits into a single commit.